### DISCUSSION TITLE
Starting text

### DISCUSSION THREAD

**Discussion by MonkePlusPlus** [FxmousKid/42Cursus-minishell#10] [Show and tell]
Add a little starting text :
![Screenshot from 2024-10-09 19-58-31](https://github.com/user-attachments/assets/1dd93608-78ab-42ff-8592-599c46bdecee)

You can modify it anytime

**Comment by FxmousKid** [comment]
:sob:

**Comment by FxmousKid** [comment]
tu as mis le text en macro ?

genre 
```C
# define ---                   ---                          -----
# deinfe /                   /                                /
# define / /               /                                /
```

ou ta fais sa comment ?

**Reply by MonkePlusPlus** [reply]
Exactement comme tu l'as dit

**Reply by FxmousKid** [reply]
vasy fait un petit screenshot ou fou le code snippet directement 

(mettre un code snippet tu fait cm sur discord (markdown en general quoi) 3 ` a la suite puis C puis 3 ` apres la derniere ligne 

```C 
```C
Code ...
puis sur une ligne appart ```
```

genre comme dans cette photo

![image](https://github.com/user-attachments/assets/cace9c97-bfc0-49a7-888c-47132289f4e9)

**Reply by MonkePlusPlus** [reply]
J'ai push tu peux le choper dans la branch exec

**Reply by FxmousKid** [reply]
c'est pas normer stp cm t dessus mtn met le sur plus de ligne, ya 2 lignes a > de 81 caracteres
d'ailleurs c'ultra haram tout sa hahahaha

**Reply by FxmousKid** [reply]
cordialement ton daron

**Reply by FxmousKid** [reply]
![image](https://github.com/user-attachments/assets/2ead7a2f-a0f6-4174-8838-5e9a5f7c0f51)

**Reply by MonkePlusPlus** [reply]
Fait moi des falafels pendant ce temps là

**Reply by FxmousKid** [reply]
mwe c sa sale raciste va

**Reply by FxmousKid** [reply]
norm moi sa vite fais avant que je remette le faite que faut normer avant de push

![image](https://github.com/user-attachments/assets/8386dda5-cbec-4496-8e6e-485294c2ac31)

**Reply by MonkePlusPlus** [reply]
c bon c fix
![Screenshot from 2024-10-09 16-50-51](https://github.com/user-attachments/assets/1f13ca59-16dc-468e-bb1b-61a00a8dff31)
